services/app-api/libs/authorization.js

```diff
@@ -6,7 +6,9 @@ import { SimpleJwksCache } from "aws-jwt-verify/jwk";
 import { SimpleJsonFetcher } from "aws-jwt-verify/https";
 
 export async function getUserDetailsFromEvent(event) {
-  await verifyEventSignature(event);
+  if (event.requestContext.accountId !== "000000000000") {
```

Comment:
Can this use `isLocalstack` or is it not a thing to share lambda and cdk code?
If it isn't we could recreate the helper or even just a comment so future devs get that it's a localstack thing.

Reply:
It would be complicated to share code between our lambdas and CDK.  For now I just recreated the isLocalStack within this function.  If we need it in another lambda, we can export it in a utility.